src/codeu/chat/RelayMain.java

```diff
@@ -54,29 +54,38 @@ public static void main(String[] args) {
 
       LOG.info("Relay object created.");
 
-      LOG.info("Loading team data...");
-
-      loadTeamInfo(relay, args[1]);
-
-      LOG.info("Done loading team data.");
-
       LOG.info("Starting relay...");
 
-      startRelay(relay, source);
+      startRelay(relay, source, args[1]);
 
     } catch (IOException ex) {
       LOG.error(ex, "Failed to establish server accept port");
     }
   }
 
-  private static void startRelay(Server relay, ConnectionSource source) {
+  private static void startRelay(final Server relay,
+                                 final ConnectionSource source,
+                                 final String teamFile) {
```

Comment:
Only 'server' and 'teamFile' actually needed to be final, but I felt it was easier to read if they were all made final. The only side-effect this creates is that you can't re-assign 'source' but it should never be re-assigned anyways.